docs/source/caching/normalized-cache.mdx

```diff
@@ -33,68 +33,53 @@ query AuthorById($id: String!) {
 
 In the above example, requesting the author of your favorite book with the `AuthorById` query will return a result from the cache if you requested your favorite book before. This works because the author is stored only once in the cache and all the fields were retrieved in the initial `BookWithAuthorName` query. If you were to request more fields, like `birthdate` for example, that wouldn't work anymore.
 
-To learn more about the process of normalization, check this [blog post](https://www.apollographql.com/blog/the-concepts-of-graphql-bc68bd819be3)
+Using a normalized cache as several advantages. The main one is to avoid storing duplicate data and save some space. Another big one is that you can use the cache as a single source of truth for your UI and react to changes in the cache. To learn more about the process of normalization, check this [blog post about cache normalization](https://www.apollographql.com/blog/apollo-client/caching/demystifying-cache-normalization/).
+
+## Memory cache
 
-## Storing your data in memory
+Apollo Android comes with a `MemoryCache` that will store your data in memory. To use it, add the `apollo-normalized-cache` artifact to your dependencies in your `build.gradle[.kts]` file:
 
-Apollo Android comes with a `MemoryCache` that will store your data in memory:
+```kotlin:title=build.gradle[.kts]
+dependencies {
+  implementation("com.apollographql.apollo3:apollo-normalized-cache:x.y.z")
+}
+```
 
-<MultiCodeBlock>
+And add it to your `ApolloClient`:
 
 ```kotlin
 // Create a 10MB MemoryCacheFactory
 val cacheFactory = MemoryCacheFactory(maxSizeBytes = 10 * 1024 * 1024)
-
 // Build the ApolloClient
 val apolloClient = ApolloClient.Builder()
   .serverUrl("https://...")
+  // normalizedCache() is an extension function on ApolloClient.Builder
   .normalizedCache(cacheFactory)
   .build()
 ```
 
-</MultiCodeBlock>
+Because the normalized cache is optional, `normalizedCache()` is an extension function on `ApolloClient.Builder()` that's defined in the `apollo-normalized-cache` artifact. It takes a `NormalizedCacheFactory` as a parameter so that it can create the cache in the background if needed.
```

Comment:
Nit: not 100% sure, is "in the background" the correct term? Maybe "on demand" or just "as needed" is more accurate?

Reply:
It's really more about not doing it on the main thread. Since it's only every creating one instance we could create it eagerly except that in most times, that would happen on the main thread. 

Maybe we just remove this sentence from the doc? And leave it for the KDoc?

Reply:
Gotcha! Wasn't sure it's what you meant, in that case it's correct 👍. Yes adding this info in the KDoc could be useful. You can remove it here or also leave it, it's doesn't hurt.